File: 485-Max Consecutive Ones/Solution.java
public class Solution {
    public int findMaxConsecutiveOnes(int[] nums) {
        int num = 0;
        int max = 0;
        for(int n:nums){
            if (n == 1){
                num++;
            }else{
                num = 0;
            }
            max = Math.max(max,num);
        }
        return max;
    }
}

File: 485-Max Consecutive Ones/Solution.py
class Solution(object):
    def findMaxConsecutiveOnes(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        num = 0
        m = 0
        for i in nums:
            if i == 1:
                num += 1
            else:
                num = 0
            m = max(num,m)
        return m